src/l10n/locales.ts

```diff
@@ -93,4 +111,22 @@ export const locales: Locales = {
     adtractionSrc: 'https://cdn.adt387.com/jsTag?ap=1589794294',
     trustpilotLocale: 'en-US',
   },
+  fr: {
+    label: 'fr',
+    iso: 'fr-FR',
+    marketLabel: 'fr',
+    marketName: 'Français',
```

Comment:
This is rather the name of the market and not the language :) Don't remember my school fre that good if it should even be _La France_ but I guess France is sufficient 🤓
```suggestion
    marketName: 'France',
```